.github/workflows/ci.yml

```diff
@@ -44,7 +34,7 @@ jobs:
 
       - uses: actions-rs/cargo@v1
```

Comment:
Not for this PR, but if we're going to maintain this ourselves going forward `actions-rs` should just be removed. It's deprecated and unmaintained _for years_.